### PR TITLE
Co.chat - add return chatlog

### PR DESCRIPTION
**Pull request by lusmoura**
## What this PR does

- Adds `return_chatlog` as an option param to co.chat
- Returns the chatlog if `return_chatlog` is set to true
- Adds unit tests for the new functionality
- Closes CNV-288

## How it was tested
- Unit tests

### CODE REVIEW

File: cohere/client.py [hunk 1]
```diff
@@ -145,15 +145,21 @@ def generate(self,
         response = self._executor.submit(self.__request, cohere.GENERATE_URL, json=json_body)
         return Generations(return_likelihoods=return_likelihoods, _future=response, client=self)
 
-    def chat(self, query: str, session_id: str = "", persona: str = "cohere", model: str = None) -> Chat:
+    def chat(self,
```
Comment (mkozakov) [inline cohere/client.py:148]: can we please add a docstring (similar to the one in summarize?)

**Review by mkozakov — APPROVED** [review]

**Review by mkozakov — CHANGES_REQUESTED** [review]
also please bump the patch version in setup.py

File: cohere/client.py (hunk 1 above)
Comment (lusmoura) [reply cohere/client.py:148]: Yep! On it

**Comment by lusmoura** [comment]
> also please bump the patch version in setup.py

I'll also do the same for another variable, should I update the patch version twice or can it wait for the next one? @mkozakov

**Comment by mkozakov** [comment]
> > also please bump the patch version in setup.py
> 
> I'll also do the same for another variable, should I update the patch version twice or can it wait for the next one? @mkozakov

If you're updating in two different PRs, update the version separately

File: setup.py [hunk 2]
```diff
@@ -24,7 +24,7 @@ def has_ext_modules(foo) -> bool:
 
 
 setuptools.setup(name='cohere',
-                 version='3.3.4',
+                 version='3.4.0',
```
Comment (mkozakov) [inline setup.py:27]: Can you please add an entry in changelog.md?

**Review by mkozakov — APPROVED** [review]

**Review by mkozakov — APPROVED** [review]

**Pull request merged**